src/include/column_inline.h

```diff
@@ -21,7 +21,8 @@ __col_insert_search_gt(WT_INSERT_HEAD *ins_head, uint64_t recno)
      *
      * Place an acquire barrier to avoid this issue.
      */
-    WT_ACQUIRE_READ_WITH_BARRIER(ins, WT_SKIP_LAST(ins_head));
+    ins = WT_SKIP_LAST(ins_head);
```

Comment:
Is this cause WT_SKIP_LAST does some dank casting under the hood?

Reply:
This is the same reasoning and `cannot take the address of an rvalue` compiler error from the other comment. WT_SKIP_LAST can also return NULL, so I figured it's best to play it safe and stick to WT_ACQUIRE_BARRIER 
```
#define WT_SKIP_LAST(ins_head) \
    (((ins_head) == NULL) ? NULL : ((WT_INSERT_HEAD *)(ins_head))->tail[0])
```